capa/main.py

```diff
@@ -780,6 +807,28 @@ def get_extractor_from_cli(args, input_format: str, backend: str) -> FeatureExtr
         raise ShouldExitError(E_CORRUPT_FILE) from e
 
 
+def get_extractor_filters_from_cli(args, input_format) -> Dict[str, Set]:
```

Comment:
please create a type for the return value, or use `typing.TypedDict` to specify the expected keys in this dict.

Reply:
```py
@dataclass
class FilterConfig:
    processes: set[int]
    functions: set[int]
    ...
```